evm-bridging/cadence/transactions/bridge_nfts_to_evm_and_wrap.cdc

```diff
@@ -0,0 +1,189 @@
+import "FungibleToken"
+import "NonFungibleToken"
+import "ViewResolver"
+import "MetadataViews"
+import "FlowToken"
+
+import "ScopedFTProviders"
+
+import "EVM"
+
+import "FlowEVMBridge"
+import "FlowEVMBridgeConfig"
+import "FlowEVMBridgeUtils"
+
+/// Bridges NFTs with provided IDs from Cadence to EVM and wraps them in a wrapper ERC721
```

Comment:
All-in-one Cadence tx to bridge NFTs from Cadence to EVM and make EVM contract calls to approve wrapper erc721 contract as operator and wrap the bridged NFTs to wrapper erc721 NFTs